.gitignore

```diff
@@ -29,3 +30,25 @@ dist-gem/
 # Ignore gulp-generated vendor files
 #
 src/stylesheets/lib
+docs/assets/fonts/
+docs/assets/js/vendor/uswds-0.9.0.js
+docs/_scss/
+docs/assets/img/alerts/
+docs/assets/img/arrow-down.png
```

Comment:
The problem exists because we don't have #1007 resolved yet. The assets for images need to exist in that directory and because there are already files in that directory, we need to ignore the files explicitly.

A fix we could do now and then revert later would be to update all the image paths into a new subdirectory of `img` for the Styleguide website and then explicitly not ignore that new subdirectory. But I figured that level of effort was better spent on resolving the issue about image paths.

Then we could just copy the files into a folder called `img/vendor` or something similar.